examples/js/loaders/EXRLoader.js

```diff
@@ -839,6 +839,10 @@ THREE.EXRLoader.prototype._parser = function ( buffer ) {
 
 			return parseV2f( dataView, offset );
 
+		} else if ( type == 'int' ) {
```

Comment:
Can you please use the strict comparison operator (`===`) in `EXRLoader`? That would be great for code style 😇

Reply:
I'll clean it up 👌